src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/ContentPresenter/ContentPresenter_Inside_ContentControl_Template.xaml

```diff
@@ -0,0 +1,107 @@
+﻿<UserControl
+		x:Class="Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.ContentPresenterPages.ContentPresenter_Inside_ContentControlTemplate"
+			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+			 xmlns:local="using:Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.ContentPresenterPages"
+			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 mc:Ignorable="d"
+			 d:DesignHeight="300"
+			 d:DesignWidth="400">
+
+	<StackPanel>
```

Comment:
There are quite a few places where we worked round this within the sources - especially in the Uno.UI.FluentTheme - can you revert those workarounds as well please? It may be possible to find those location by searching for the issue ids.

E.g. 

https://github.com/unoplatform/uno/blob/9be0f64372ebd97f7dda6d4144d265d170c8d608/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/Button_themeresources.xaml#L139

Reply:
This also fixes https://github.com/unoplatform/uno/issues/6452 right?